book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+
+**Volunteers**
+Volunteers were an essential part of the day, and our event would not have been successful without their help! 
+For clear communication with volunteers, we:
+- **provided an availability form** for volunteers to fill in, indicating specific sessions or hours they could help, what they would like to do, and their own accessibility requirements. 
+- **hosted an on-boarding call** to talk through the agenda and highlight the parts of the day where we would require help from volunteers.
+- **created a dedicated volunteer's Slack channel**, so that volunteers could ask questions relating to the event, and we could easily deal with any trouble shooting during the event.
+- [**created a final volunteer's task form**](), so that all volunteers had been assigned tasks before the event began 
+
+
+(er-datahazardscasestudy-organising-catering)=
+### Catering 
+For our event we choose a local, entirely vegan caterer, to align with the core values of the event. 
+To ensure successful ordering and delivery of the catering on the day we: 
+- estimated the number of attendees to provide catering - including volunteers, speakers and organisers. 
+- recieved a quote from the caterers in advance and ensure we had the budget to cover the quote.
+- collected dietary requirements to ensure that all attendees were catered for, and ensured the catering company recieved these requirements in advance. 
+- arranged a delivery time and location.
+- ensured payment to catering company as per company and organisation policies.
+
+Tip: make sure terms and conditions are read and timing requirements from different parties involved are clear from the beginning. It might be that the caterer you are ordering from requires a purchase order to be completed a certain time in order to complete the order action.
+
+
+(er-datahazardscasestudy-hosting)=
+## Hosting the Event 
+There is already a chapter on {ref}`cl-organising-conferences` which includes an in-depth dive into requirements for accessibility. 
+Likewise there is also a section for {ref}`cm-comms-overview-accessibly`.
+As well as on {ref}`cl-hybrid-collab-guidelines`. 
+
+From our experience, here are some extra tips on the day of the event.
+
+(er-datahazardscasestudy-hosting-startingevent)=
+### Starting the Event 
+- As the hosts arrive in-person at least 1 hour prior to start time to ensure the zoom link and audio-visual equipment (speakers, screens and microphones) are all working.
+- Open the online call early to allow for: 
+    - volunteers to settle in and open any breakout-rooms.
+    - participants to join in advance and set-up break out rooms for networking events and workshops later in the day.
+
+(er-datahazardscasestudy-hosting-hybridevent)=
+### Hybrid Event Tips
+We chose to host the Symposium in a hybrid format as this would allow attendance for those interested in the topic, and speakers who would be unable to attend in-person.
```

Comment:
Symposium is variously capitalised and not throughout the document, make sure it is consistent